t/plans.t

```diff
@@ -0,0 +1,106 @@
+=pod
+
+=encoding utf-8
+
+=head1 PURPOSE
+
+Tests for various plans
+
+=cut
+
+use v5.14;
+use autodie;
+use utf8;
+use feature "state";
+use Test::Modern;
+use Test::Exception;
+use Digest::SHA qw(sha1_hex);
+
+use Attean;
+use Attean::RDF;
+
+use Attean::Plan::Iterator;
+
+my $ci = Attean::CodeIterator->new(
+											  generator => sub {
+												  state $i = 0;
+												  return undef if ($i > 2);
```

Comment:
This is never actually executed, right? Seems like a strange setup since `$i` is never incremented.